### DISCUSSION TITLE
connecting several CRSF to PWM converters to a single ELRS receiver

### DISCUSSION THREAD

**Discussion by mstrens** [ExpressLRS/ExpressLRS#1399] [Q&A]
I flight quite big gliders (5m) with many servos. I do not use flight controllers.
Currently I am using Frsky receiver and Sbus because it simplifies the wiring/connecting
I am looking to replace Frsky RF link by ELRS (e.g. a happymodel 24TX pro and a Matek Rx with diversity).

I would like to know, if it is possible to connect several Matek CRSF to PWM converters to the same ELRS receiver in a daisy chain (like for Sbus).
Please note that I plan to connect only the receiver "Tx" pin to the Rx pin of each of the converter.
The Tx pin of the converters would not be connected to the receiver because I plan to connect it to an arduino (pro mini) that would provide all telemetry data (GPS, altitude, voltage, ...) in a CRSF format.

Is it possible to connect several converters in a daisy chain if the TX pin of the converters are not connected?
How long can be the wire between receiver and converters? I expect that the length would be less than with Sbus because the baudrate of CRSF is about 400k instead of 100k for Sbus.

**Comment by mstrens** [comment]
I had a contact with the person that developed (and put on github) a CRSF to PWM software running e.g. on a bluepill board.
The concept seems ok.
Still I am not sure that the signals can support a baudrate of 400K on quite long distance. Furthemore, it seems that the Matek devices require a GPS. This does not make sense for this use case.
So it would require to change the firmware but it is not open source.

**Reply by secretman28** [reply]
no the matek doent need a gps, if no gps it just flash the led all work as it should, im planning to do the same for a big turbien jet on the bench all seam to work good i luv to have only 1 plug of 3 wire ( 2 bigger for battery feed and one tx to the matek rx) on each wing, and not have to plus 5 plug to put 1 wing 

my concern is, if only matek can allow us to remap all ch output, they put that you can change only the ch5 output to any of the 16ch avail

some one is good to build a custom firmware? they already did the remap part just need to add all of the oter output, in the cli anything other than 5 repport an error
